[SDT64B] modified the UART pin number.

hjpark-sdt opened the pull request:
### Description
I'm heejung Park in Sigma Delta Technologies.
Please check below.

1. Changed pin number of UART0 to UART2 because we made a new hardware of SDT64B.
2. For normal operation of UART2(PTD8, PTD9, PTD10, PTD11), added _UART_5_ in PeripheralNames.h.

Thanks.

<!-- 
    Required
    Add here detailed changes summary, testing results, dependencies 
    Good example: https://os.mbed.com/docs/latest/reference/workflow.html (Pull request template)
-->


### Pull request type

<!-- 
    Required
    Please add only one X to one of the following types. Do not fill multiple types (split the pull request otherwise).
    Please note this is not a GitHub task list, indenting the boxes or changing the format to add a '.' or '*' in front
    of them would change the meaning incorrectly. The only changes to be made are to add a description text under the
    description heading and to add a 'x' to the correct box.
-->
    [ ] Fix
    [ ] Refactor
    [x] Target update
    [ ] Functionality change
    [ ] Docs update
    [ ] Test update
    [ ] Breaking change

### Reviewers

<!-- 
    Optional
    Request additional reviewers with @username
-->

Commit:
first

0xc0170 reviewed (changes requested):
Please fix the commit message (include the details from the description here)

I would also recommend split the changes to separate commits (adding bootloader support and fixing serial pins)

hjpark-sdt commented:
> ### Description
> I'm heejung Park in Sigma Delta Technologies.
> Please check below.
> 
> 1. Changed pin number of UART0 to UART2 because we made a new hardware of SDT64B.
> 2. For normal operation of UART2(PTD8, PTD9, PTD10, PTD11), added _UART_5_ in PeripheralNames.h.
> 
> Thanks.
> 
> ### Pull request type
> ```
> [ ] Fix
> [ ] Refactor
> [x] Target update
> [ ] Functionality change
> [ ] Docs update
> [ ] Test update
> [ ] Breaking change
> ```
> ### Reviewers

Okay. I split
